package.json

```diff
@@ -43,6 +43,7 @@
     "chai": "^3.5.0",
     "enzyme": "^2.3.0",
     "jsdom": "^9.4.0",
+    "lodash": "4.14.1",
```

Comment:
Adding this causes the tests to break due to imports. To replicate, delete your node_modules folder and run `npm run test`. Is there a reason why this has been added?